app/src/docs/components/VisualEditorPage/codeSnippets.js

```diff
@@ -0,0 +1,122 @@
+export const JavaModule =
```

Comment:
The snippets would look better without the start and end newlines in my opinion... Or is it designed like this?

Reply:
hmm, I'll take another look and perhaps add the line numbers back in while I'm at it. 

https://share.goabstract.com/3fe2aa03-3291-425f-ad69-4fb0e191a670 - shows some appetite for top/bottom margin at least.

Reply:
Yep, lots of padding :) You could get rid of the extra newlines and just use padding.

Reply:
sounds good 👍